bibliopixel/drivers/SPI/base.py

```diff
@@ -6,7 +6,18 @@
 
 class SPIBase(DriverBase):
     """Base driver for controling SPI devices on systems like the Raspberry Pi
-       and BeagleBone"""
+      and BeagleBone
+
+    Provides the same parameters of
+    :py:class:`bibliopixel.drivers.driver_base.DriverBase` as
+    well as those below:
+
+    :param str dev: SPI device path
+    :param str spi_interface: Interface API with which to connect to the SPI
+        device. One of :py:class:`bibliopixel.drivers.spi_interfaces.SPI_INTERFACES`
```

Comment:
Ah, I got stuck on reading this sentence because I expected a verb to appear. 

What about:  `Interface API with which to connect to the SPI device: must be one of the entries in...`?